inc/front/minify.php

```diff
@@ -139,7 +139,7 @@ function rocket_insert_minify_js_in_footer() {
 function rocket_concatenate_google_fonts( $buffer ) {
 	// Get all Google Fonts CSS files.
 	$buffer_without_comments = preg_replace( '/<!--(.*)-->/Uis', '', $buffer );
-	preg_match_all( '/<link\s*.+href=[\'|"](.+fonts\.googleapis\.com.+)(\'|").+>/iU', $buffer_without_comments, $matches );
+	preg_match_all( '/<link\s*.+href=[\'|"](.+fonts\.googleapis\.com.+)[\'|"].+>/iU', $buffer_without_comments, $matches );
```

Comment:
This regex pattern is flawed.
Here are some results with sadistic html (you know me): https://regex101.com/r/oK8QgH/2.
The main problems are `['|"]` (the `|` character shouldn't be here) and the `.+` at the end (mandatory character before the closing `>`).
Here is my suggestion: `<link(?:\s.+)?\shref\s*=\s*('|")((?!\1).+)\1(?:\s.*)?>` https://regex101.com/r/kTP2Yo/2.
It may seem a bit overkill, especially since I use a back reference to match the same character for the tag delimiter (" or '). Be aware: YOU HAVE A NEW MATCHING GROUP.

Reply:
it needs to match fonts.googleapis.com only though

Reply:
Indeed, but the 2 problems I pointed are still valid:
`<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato|Open+Sans:400,700|Raleway:500i">` <= Got 2 pipes here, and no extra characters at the end between `"` and `>`.

My pattern can be simplified like this:
`<link(?:\s.+)?\shref\s*=\s*['"](.+)['"](?:\s.*)?>`